src/shared/dtos/pagination-params.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { IsNumber, IsOptional } from 'class-validator'
+import { Transform } from 'class-transformer'
+import { ApiPropertyOptional } from '@nestjs/swagger'
+
+export class PaginationParamsDto {
```

Comment:
One small opinion may be better to name `PaginationDto`, We are using it as query params that can be not part of its name.

Reply:
@asad-mlbd 
You mean since we use it as query params, the work `params` should not be included? 🤔